src/btree-psi.scm

```diff
@@ -27,7 +27,7 @@
 
 (use-modules (opencog))
 (use-modules (opencog query))  ; XXX work-around relex2logic bug
-
+(load-from-path "time-map.scm") ;;; octomap for faces
```

Comment:
In the long run, this should be modularized, i.e. `(use-modules (opencog time-map))`